README.md

```diff
@@ -13,10 +13,31 @@ In the absence of bugs, it is expected that after consuming the BEL
 connections and TCL constraints Vivado will produce a bitstream identical to
 the bitstream the FASM file would generate.
 
+Installing
+----------
+
+Before cloning this repo, the RapidWright and capnproto-java repos must be installed first.
+
+Follow the instructions to install RapidWright [here](https://github.com/Xilinx/RapidWright) and clone capnproto-java [here](https://github.com/capnproto/capnproto-java).
+
+After RapidWright and capnproto-java are installed, clone this repo and run the following commands:
+  - `make env` - python >=3.7 is required - you may have to upgrade and then change Makefile to use the new version (same for Invoking below)
+    - To get python3.7 on Ubuntu 16.04 (or above) do the following:
+      - sudo apt update
+      - sudo apt install software-properties-common
+      - sudo add-apt-repository ppa:deadsnakes/ppa
```

Comment:
We shouldn't need `python3.7`? The tool should work with `python3.7`, `python3.8`, `python3.9` (and we should test it).